packages/deprecated-component-library-helpers/styles/type.scss

```diff
@@ -0,0 +1,359 @@
+}
+
+@mixin ca-type-ideal-small($size: 14, $args...) {
+  @include ca-type-ideal($size: $size, $weight: $ca-weight-light, $args...);
+}
+
+@mixin ca-type-ideal-small-bold($size: 14, $args...) {
+  @include ca-type-ideal($size: $size, $weight: $ca-weight-book, $args...);
+}
+
+@mixin ca-type-ideal-notification($size: 15, $args...) {
+  @include ca-type-ideal(
+    $size: $size,
+    $weight: $ca-weight-light,
+    $line-height-in-grid-units: 3/4,
+    $args...
+  );
+}
+
+@mixin ca-type-ideal-label($size: 12, $args...) {
+  $letter-spacing-in-px: 0.5;
+  @include ca-type-ideal(
+    $size: $size,
+    $weight: $ca-weight-medium,
+    $letter-spacing: $letter-spacing-in-px / $size * 1em,
+    $args...
+  );
+  text-transform: uppercase;
+}
+
+// Deprecated, use ca-type-ideal-label instead.
+@mixin ca-type-ideal-labels-and-legends($args...) {
+  @include ca-type-ideal-label($args...);
+}
+
+@mixin ca-type-ideal-control-action($size: 16, $args...) {
+  @include ca-type-ideal($size: $size, $weight: $ca-weight-book, $args...);
+  text-decoration: none;
+  text-decoration-skip-ink: auto;
+  color: $kz-color-cluny-500;
+  cursor: pointer;
+  &:hover {
+    text-decoration: underline;
+  }
+}
+
+@mixin ca-type-ideal-button($size: 18, $args...) {
+  @include ca-type-ideal($size: $size, $weight: $ca-weight-medium, $args...);
+}
+
+// Greycliff styles
+
+// A utility to strip the unit from a value. Eg: `1.5rem` to `1.5`
+@function strip-unit($value) {
+  @return $value / ($value * 0 + 1);
+}
+
+// Given a rem unit like `1.5rem`, convert to pixels, but without the unit,
+// (eg `24`). This is to maintain backwards compatibility with existing usages
+// of the mixins, which are specified in unitless pixels. Eg: `ca-type-ideal-heading(32)`
+// will become `kz-typography-heading-4(32)` which would be converted internally to `2rem`.
+
+@function to-unitless-pixels($value-in-rems) {
+  @return strip-unit($value-in-rems) * 16;
+}
+
+@function to-grid-units($value-in-rems) {
+  @return $value-in-rems / $ca-grid;
+}
+
+@mixin kz-typography-greycliff(
+  $size,
+  $weight,
+  $line-height-in-grid-units: 1,
+  $letter-spacing: $kz-typography-heading-1-letter-spacing,
+  $args...
+) {
+  @include ca-type(
+    $size-ratio: $size / 16,
+    $family: $kz-typography-heading-1-font-family,
+    $base-size: $ca-greycliff-font-base-size,
+    $descender-height: $ca-greycliff-font-descender-height,
+    $line-height-in-grid-units: $line-height-in-grid-units,
+    $letter-spacing: $letter-spacing,
+    $weight: $weight,
+    $args...
+  );
+}
+
+@mixin kz-typography-display-0(
+  $size: to-unitless-pixels($kz-typography-display-0-font-size),
+  $args...
+) {
+  @include kz-typography-greycliff(
+    $size: $size,
+    $weight: $kz-typography-display-0-font-weight,
+    $letter-spacing: $kz-typography-display-0-letter-spacing,
+    $line-height-in-grid-units:
+      to-grid-units($kz-typography-display-0-line-height),
+    $args...
+  );
+}
+
+@mixin kz-typography-heading-1(
```

Comment:
Also note: there's no need to put "greycliff" in the name like we did for "ideal", because the `kz` prefix means it can't clash with mixins like `ca-type-heading`